cli/src/clients.rs

```diff
@@ -16,7 +16,7 @@ pub async fn create_trunk_repo(
     origin: &str,
     api_token: &str,
     org_slug: &str,
-    repo: &Repo,
+    repo: &RepoUrlParts,
```

Comment:
Nit, update the parameter to match if we stick with this new name type.
```suggestion
    repo_url_parts: &RepoUrlParts,
```